lib/Service/DatabaseService.php

```diff
@@ -102,6 +103,27 @@ public function getTokenInfoFromDB(string $token): array {
 		];
 	}
 
+	/**
+	 *
+	 * Get All token information from database
+	 *
+	 * @throws Exception
+	 */
+	public function deleteAllExpiredTokenFromDB(): void {
```

Comment:
```suggestion
	public function deleteExpiredTokens(): void {
```
as this is a DB class, its clear that its deleting from the DB